### PR TITLE
Added translations for chineese language.

### PR DESCRIPTION
**Pull request by alfidinouhail**
> Q	A
> Branch?	3.4
> Bug fix?	yes
> New feature?	no
> BC breaks?	no
> Deprecations?	no
> Tests pass?	yes
> Fixed tickets	#30447
> License    MIT

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf [hunk 1]
```diff
@@ -310,10 +310,30 @@
...
+                <source>This is not a valid Business Identifier Code (BIC).</source>
+                <target>这不是有效的业务标识符代码（BIC）。</target>
+            </trans-unit>
             <trans-unit id="82">
                 <source>Error</source>
                 <target>错误</target>
             </trans-unit>
+            <trans-unit id="83">
+                <source>This is not a valid UUID.</source>
+                <target>这不是有效的UUID</target>
```
Comment (fabpot) [inline src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf]: missing dot at the end

File: src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf [hunk 2]
```diff
@@ -310,10 +310,30 @@
...
+            </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>此值应为 {{compared_value}} 的倍数</target>
```
Comment (fabpot) [inline src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf]: missing dot at the end

File: src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf [hunk 3]
```diff
@@ -310,10 +310,30 @@
                 <source>This value does not match the expected {{ charset }} charset.</source>
                 <target>该值不符合 {{ charset }} 编码。</target>
             </trans-unit>
+            <trans-unit id="81">
+                <source>This is not a valid Business Identifier Code (BIC).</source>
+                <target>这不是有效的业务标识符代码（BIC）。</target>
```
Comment (fabpot) [inline src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf]: Not sure, but it looks like there is an extra space before the dot.

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf (hunk 2 above)
Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf]: Please, maintain the white spaces in the placeholder name: `{{compared_value}}` should be `{{ compared_value }}`   Thanks!

File: src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf [hunk 4]
```diff
@@ -310,10 +310,30 @@
...
+                <source>This is not a valid UUID.</source>
+                <target>这不是有效的UUID</target>
+            </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>此值应为 {{compared_value}} 的倍数</target>
+            </trans-unit>
+            <trans-unit id="85">
+                <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
+                <target>此业务标识符代码（BIC）与IBAN {{iban}} 无关。</target>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf]: Here we must also keep the white spaces: `{{iban}}` should be `{{ iban }}`  Thanks!

File: src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf (hunk 3 above)
Comment (ro0NL) [reply src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf]: that space is part of `U+FF09 fullwidth right parenthesis`

https://en.wikipedia.org/wiki/Chinese_punctuation#Marks_similar_to_European_punctuation

File: src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf (hunk 2 above)
Comment (alfidinouhail) [reply src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf]: I did the changes. Thanks.

File: src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf (hunk 4 above)
Comment (alfidinouhail) [reply src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf]: Hello, I did the changes. Thanks.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @alfidinouhail.

**Comment by nicolas-grekas** [comment]
@alfidinouhail we are looking for help to translate the new Symfony 5 book written by @fabpot into Chinese. Do you know if some Chinese-native people could be interested in doing so?
See https://www.kickstarter.com/projects/fabpot/symfony-5-the-fast-track for more info :)

**Pull request merged**